--- gmetadom-0.0.3/configure.in	Thu Mar 28 16:37:30 2002
+++ gmetadom-0.0.3-/configure.in	Mon May 20 00:28:23 2002
@@ -154,7 +154,7 @@
 AC_SUBST(GDOME_CFLAGS)
 AC_SUBST(GDOME_LIBS)
 
-AC_LANG_CPLUSPLUS
+AC_LANG_PUSH([C++])
 AC_MSG_CHECKING(whether iconv has a const argument)
 AC_TRY_COMPILE(
         [#include <iconv.h>],
@@ -171,7 +171,7 @@
                 AC_MSG_RESULT(no)
         ]
 )
-AC_LANG_RESTORE
+AC_LANG_POP([C++])
 
 AC_CHECK_PROG(HAVE_ICONV, iconv, yes, no)
 if test $HAVE_ICONV = "yes"; then
@@ -298,15 +298,6 @@
 xml/Makefile
 src/Makefile
 src/shared/Makefile
-dnl src/abstract_cpp_heap/Makefile
-dnl src/abstract_cpp_heap/abstract_cpp_heap.conf
-dnl src/abstract_cpp_heap/xsl/Makefile
-dnl src/abstract_cpp_heap/include/Makefile
-dnl src/abstract_cpp_heap/basic/Makefile
-dnl src/abstract_gdome_cpp_heap/Makefile
-dnl src/abstract_gdome_cpp_heap/xsl/Makefile
-dnl src/abstract_gdome_cpp_heap/include/Makefile
-dnl src/abstract_gdome_cpp_heap/core/Makefile
 src/gdome_cpp_smart/Makefile
 src/gdome_cpp_smart/gdome_cpp_smart.conf
 src/gdome_cpp_smart/xsl/Makefile
diff -ur gmetadom-0.0.3-/src/gdome_cpp_smart/doc/Makefile.am gmetadom-0.0.3/src/gdome_cpp_smart/doc/Makefile.am
--- gmetadom-0.0.3-/src/gdome_cpp_smart/doc/Makefile.am	Mon Dec  3 15:02:31 2001
+++ gmetadom-0.0.3/src/gdome_cpp_smart/doc/Makefile.am	Mon May 20 00:51:39 2002
@@ -28,7 +28,7 @@
 #  DocumentEvent.html \
 #  UIEvent.html
 #  MouseEvent.html \
-#  EventCapturer.html \
+#  EventCapturer.html 
 
 noinst_DATA = \
   index.html \
diff -ur gmetadom-0.0.3-/src/gdome_caml/test/Makefile.am gmetadom-0.0.3/src/gdome_caml/test/Makefile.am
--- gmetadom-0.0.3-/src/gdome_caml/test/Makefile.am	Tue Mar 26 14:36:50 2002
+++ gmetadom-0.0.3/src/gdome_caml/test/Makefile.am	Mon May 20 00:54:36 2002
@@ -38,8 +38,8 @@
   $(top_builddir)/src/gdome_caml/mlgdome.cmxa \
   $(top_builddir)/src/gdome_caml/ocaml/mlogdome.cmxa
 
-test : test.ml
+test$(EXEEXT) : test.ml
 	$(OCAMLC) -o $@ $(OCAML_FLAGS) -cclib "$(GDOME_LIBS)" gdomeInit.cmo $(C_LIBS) $(OCAML_LIBS) $<
 
-test.opt : test.ml
+test.opt$(EXEEXT) : test.ml
 	$(OCAMLOPT) -o $@ $(OCAML_FLAGS) -cclib "$(GDOME_LIBS)" gdomeInit.cmx -cclib "$(C_LIBS)" $(OCAMLOPT_LIBS) $<
