--- gmetadom-0.2.3/src/gdome_cpp_smart/include/GdomeSmartDOMNode.hh.orig	2005-07-08 11:53:05.000000000 +0200
+++ gmetadom-0.2.3/src/gdome_cpp_smart/include/GdomeSmartDOMNode.hh	2006-01-29 18:58:16.236925000 +0100
@@ -36,7 +36,7 @@
     Node(const Node&);
   ~Node();
 
-  Node::Node(const class EventTarget&);Node& operator=(const Node&);
+  Node(const class EventTarget&);Node& operator=(const Node&);
   bool operator==(const Node& obj) const { return gdome_obj == obj.gdome_obj; }
   bool operator!=(const Node& obj) const { return !(*this == obj); }
   
--- gmetadom-0.2.3/src/gdome_cpp_smart/include/GdomeSmartDOMEventTarget.hh.orig	2005-07-08 11:53:06.000000000 +0200
+++ gmetadom-0.2.3/src/gdome_cpp_smart/include/GdomeSmartDOMEventTarget.hh	2006-01-29 18:58:38.614323500 +0100
@@ -36,7 +36,7 @@
     EventTarget(const EventTarget&);
   ~EventTarget();
 
-  EventTarget::EventTarget(const class Node&);EventTarget& operator=(const EventTarget&);
+  EventTarget(const class Node&);EventTarget& operator=(const EventTarget&);
   bool operator==(const EventTarget& obj) const { return gdome_obj == obj.gdome_obj; }
   bool operator!=(const EventTarget& obj) const { return !(*this == obj); }
   
