diff -Nru gmetadom-0.1.6.orig/src/gdome_caml/Makefile.am gmetadom-0.1.6/src/gdome_caml/Makefile.am
--- gmetadom-0.1.6.orig/src/gdome_caml/Makefile.am	Thu Mar 13 11:01:31 2003
+++ gmetadom-0.1.6/src/gdome_caml/Makefile.am	Thu Mar 20 19:23:09 2003
@@ -63,7 +63,6 @@
 	$(OCAMLMKLIB) -o $(ARCHIVE) $(GDOME_LIBS) $(CMXA_DEPS)
 
 $(DLL) $(ARCHIVE).a: $(LO_S)
-	# use .lo for ocamlmklib
 	for f in $(LO_S); do	\
 		cp $$f $$f.o;	\
 	done
